### DISCUSSION TITLE
Moti Pressable OnPress

### DISCUSSION THREAD

**Discussion by karmatradeDev** [nandorojo/moti#239] [Q&A]
situation: i have a modal and it has a motiPressable as a child when i click the pressable buttons behind it gets clicked that is behind the modal. i usually fix this problem by using e.stopPropagation in onPress method of pressable. motiPressable doesnt have an event param in its on press function. has anyone encountered a work around? i'm convinced i/m missing something.

**Comment by nandorojo** [comment]
it’s possible that the event isn’t getting passed down properly. what platform is this issue on?

**Reply by karmatradeDev** [reply]
ios safari. not native everywhere else it works great
